x-parser.js

```diff
@@ -382,6 +377,7 @@ export class XParser {
       case XParser.#danglingQuote: return XParser.#try(string, stringIndex,
         XParser.#startTagSpaceMalformed);
       case XParser.#startTagSpace: return XParser.#try(string, stringIndex,
+        XParser.#startTagSpaceMalformed,
```

Comment:
Again, this is the new _invalid_ state transition that was added to decrease the complexity of the `#startTagSpace` pattern.